### DISCUSSION TITLE
Enable/Disable buttons via MQTT

### DISCUSSION THREAD

**Discussion by paducz** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#686] [Q&A] [answered]
It is possible to disable/enable the SPA buttons via MQTT? This option is available in the web app

**Comment by visualapproach** [comment]
No, only via web interface. Would you benefit from setting it from MQTT?

**Reply by paducz** [reply]
Yes - I am renting a house with spa on airbnb and I would like to disable bubbles button after 11pm to not disturb my neighbours. Bubbles are quite loud. It would be great to setup automation in home assistant to achieve this

**Reply by visualapproach** [reply]
Ok I'll look at it

**Reply by paducz** [reply]
Thank you very much

**Reply by visualapproach** [reply, marked as answer]
@paducz dev branch updated. I only made some basic testing but you can now configure things from MQTT. New switches by autodiscovery. Update, restart and wait a few minutes.

**Reply by paducz** [reply]
Awesome, thanks again!
